Eigenvalues/InversePowerMethod.m

```diff
@@ -0,0 +1,32 @@
+% [USES] Ad-hoc/SST.m
+% [USES] Gauss/G/G.m
+function [lambda, y, step] = InversePowerMethod(A, tol, max_iter, miu)
+    [n m] = size(A);
```

Comment:
un comentariu deasupra de linia asta: "check if it's a square matrix, otherwise there won't be any eigenvalues"